docs/Instance-Variable-Assumption.md

```diff
@@ -18,30 +18,30 @@ class Foo
 end
 ```
 
-Bad:
+Good as well:
 
-```ruby
+```Ruby
 class Foo
-  def go_foo!
-    @bar = :foo
+  def foo?
+    bar == :foo
   end
 
-  def foo?
-    @bar == :foo
+  def bar
+    @bar ||= :foo
   end
 end
 ```
 
-Good:
+Bad:
 
-```ruby
+```Ruby
 class Foo
-  def foo?
-    bar == :foo
+  def go_foo!
+    @bar = :foo
   end
 
-  def bar
-    @bar ||= :foo
+  def foo?
+    @bar == :foo
   end
 end
```

Comment:
The diff above is hard to read. I basically just switched the two examples around, that's all.